scripts/misc/commit_message_recom.txt

```diff
@@ -0,0 +1,34 @@
+#################################################
+#################################################
+Summarize changes in around 50 characters or less.
+
+More detailed explanatory text. Wrap it to about 72
+characters or so. In some contexts, the first line is treated as the
+subject of the commit and the rest of the text as the body. The
+blank line separating the summary from the body is critical (unless
+you omit the body entirely); various tools like `log`, `shortlog`
+and `rebase` can get confused if you run the two together.
+
+Explain the problem that this commit is solving. Focus on why you
+are making this change as opposed to how (the code explains that).
+Are there side effects or other unintuitive consequences of this
+change? Here's the place to explain them.
+
+Further paragraphs come after blank lines.
+
+- Bullet points are okay, too.
+
+- Typically a hyphen or asterisk is used for the bullet, preceded
+  by a single space, with blank lines in between, but conventions
+  vary here.
+
+All commits need to reference a ticket, again separated by a blank
+line from the commit message summary above:
+
+Resolves: ABC-111, #123
```

Comment:
It's required? What about CRs from external guys if it's required?

Reply:
good point, comment was adjusted accordingly.